tests/Unit/ControlSystem/Test_UpdateFunctionOfTime.cpp

```diff
@@ -0,0 +1,228 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "ControlSystem/UpdateFunctionOfTime.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/PiecewisePolynomial.hpp"
+#include "Domain/FunctionsOfTime/RegisterDerivedWithCharm.hpp"
+#include "Domain/FunctionsOfTime/SettleToConstant.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Framework/ActionTesting.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+template <typename Metavariables>
+struct TestSingleton {
+  using chare_type = ActionTesting::MockSingletonChare;
+  using array_index = size_t;
+  using metavariables = Metavariables;
+  using get_const_global_cache_tags = tmpl::list<>;
+  using mutable_global_cache_tags = tmpl::list<domain::Tags::FunctionsOfTime>;
+  using phase_dependent_action_list =
+      tmpl::list<Parallel::PhaseActions<typename metavariables::Phase,
+                                        metavariables::Phase::Initialization,
+                                        tmpl::list<>>>;
+};
+
+struct TestingMetavariables {
+  enum class Phase { Initialization, Testing, Exit };
+  using component_list = tmpl::list<TestSingleton<TestingMetavariables>>;
+};
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.UpdateFunctionOfTime",
+                  "[Unit][ControlSystem]") {
+  INFO("Test updating FunctionOfTime in global cache.");
```

Comment:
Remove this.  It isn't scoped so it will display for any failure, and it doesn't say anything the test name doesn't.